specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -1913,6 +1913,84 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/containerlogs": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets the last lines of docker logs for the given site",
```

Comment:
did you want to add an appropriate `produces` element? (see [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) )

``` json
 "produces": [
     "text/plain"
  ],
```

Reply:
Updated

Reply:
Please let me know if I'm missing something here but this endpoint does seem to work for me:
```
POST https://management.azure.com/subscriptions/ffa52f27-be12-4cad-b1ea-c2c241b6cceb/resourceGroups/linux-test-appgroup/providers/Microsoft.Web/sites/linux-test-app?api-version=2016-08-01
405 Method Not Allowed 749.00 ms
{
"error": {
"code": "HttpMethodIsNotSupported",
"message": "The HTTP method 'POST' is not supported."
}
}
```

Reply:
The deployment to production happened two weeks ago. I have confirmed it working with my testa pp. Can you please try again?

Reply:
Seems like the endpoint is returning a base 64 encoded content - in this case, the return type shouldn't be `file` but `byte`. The `file` response type should be unencoded (gzipped okay). 

Also, the server is actually returning me `application/json` for content type.